pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 79
-target-version = ['py39', 'py310', 'py311', 'py312']
+target-version = ['py310', 'py311', 'py312']
```

Comment:
@c0state Removal of python versions may necessitate a version bump, as `fourmat` is meant to be consumed as a lib